setup/instance/inputs.go

```diff
@@ -56,12 +56,12 @@ Layer0 API will use its own key with limited permissions to provision AWS resour
 
 const INPUT_AWS_REGION_DESCRIPTION = `
 AWS Region: The region input variable specifies which region to provision the
-AWS resources required for Layer0. The following regions can be used: 
+AWS resources required for Layer0. Regions that support the following services can be used:
```

Comment:
This sentence is grammatically awkward and it makes the prompt non-intuitive. It almost makes it look like you want the user to enter a service, not a region.

Reply:
Which sentence is grammatically awkward... I'm assuming the second one? This is copy & paste from @sesh-kebab 's previously merged patch. I do agree that if you're not reading it carefully, you might think one of the bulleted items should be an option for this prompt. However, I don't think we should hardcode regions that have the services listed available here.

Reply:
Or maybe another possible sentence is "Regions where the following services are allowed may be entered".

Reply:
Still, the form of this input prompt differs slightly from other prompts to a degree where you're listing services underneath and someone squinting might naturally think they have to enter a service.

Reply:
I would prefer to table this for now and see if anyone else gets tripped up on it during testing. To me, the problem lies in that if you separate out those services in a list, it will seem like maybe they should be entered into the prompt here. But if you don't put those services into a list, it makes for a longer, weirder sentence that would likely be ignored.